pageserver/src/pgdatadir_mapping.rs

```diff
@@ -2085,7 +2085,7 @@ impl DatadirModification<'_> {
                 self.tline.aux_file_size_estimator.on_add(content.len());
                 new_files.push((path, content));
             }
-            (None, true) => warn!("removing non-existing aux file: {}", path),
+            (None, true) => info!("removing non-existing aux file: {}", path),
```

Comment:
Please add a comment explaining when this can happen.

Reply:
Done